src/Bridge/Doctrine/Common/Filter/DateFilterTrait.php

```diff
@@ -79,4 +80,20 @@ protected function getFilterDescription(string $property, string $period): array
             ],
         ];
     }
+
+    /**
+     * Normalize the value.
+     */
+    private function normalizeValue($value, string $operator): ?string
```

Comment:
I might be picky on this, but wouldn't it be better (semantically wise) to use early returns?

First exclude the bad scenario and then only return the happy scenario ?

```suggestion
    private function normalizeValue($value, string $operator): ?string
    {
            if (false === \is_string($value)) {
               $this->getLogger()->notice('Invalid filter ignored', [
                   'exception' => new InvalidArgumentException(sprintf('Invalid value for "[%s]", expected string', $operator)),
               ]);

               return null;
            }

            return $value;
    }
```

Reply:
Using a guard clause reduces the cyclomatic complexity and makes the code more readable (imho). I would keep this block as is.

Reply:
But this is what I'm trying to say here :-)

First the guard (_the unhappy scenario_), then relevant returned `$value` (_the happy scenario_).

Reply:
I'm :+1: for @drupol's suggestion.

Reply:
Let's not forget this thing :-)

Reply:
Oh it's done already ! Great :-)